git-hub

```diff
@@ -507,8 +510,8 @@ class RequestManager:
 # methods get bind with the last value of method ('delete') in this case, which
 # is not only what we want, is also very dangerous.
 def make_method(method):
-    return lambda self, url, *args, **kwargs: \
-        self.json_req(url, method, *args, **kwargs)
+    return lambda self, url, media_type=None, *args, **kwargs: \
```

Comment:
Maybe can change this to `media_type= "application/vnd.github.v3+json"`. 
That might be the best place to set the default.

Reply:
Sounds like the right thing to do. I'll update it